specification/deploymentmanager/resource-manager/Microsoft.DeploymentManager/preview/2019-11-01-preview/deploymentmanager.json

```diff
@@ -0,0 +1,2465 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-11-01-preview",
+    "title": "AzureDeploymentManager",
+    "description": "REST APIs for orchestrating deployments using the Azure Deployment Manager (ADM). See https://docs.microsoft.com/en-us/azure/azure-resource-manager/deployment-manager-overview for more information."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DeploymentManager/serviceTopologies/{serviceTopologyName}": {
```

Comment:
all top-level entities (serviceTopologies, steps, and rollouts) are missing the subscription-level LIST operations. https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#request---get-resource-collection-under-subscription

Reply:
We are now in preview and we'll have these added in future versions as we approach GA.

Reply:
I will sign off, but this will block stable release of the API. Please add in the next update.

Reply:
Of course, we'll have this planned for the next release. thank you!